Fill.py

```diff
@@ -509,7 +509,8 @@ def mark_for_locking(location: Location):
         if progitempool:
             raise FillError(
                 f"Not enough locations for progression items. "
-                f"There are {len(progitempool)} more progression items than there are available locations.",
+                f"There are {len(progitempool)} more progression items than there are available locations.\n"
+                f"Unfilled locations: {multiworld.get_unfilled_locations()}.",
```

Comment:
A few places split these into separate lines, which I think looks better as well
```suggestion
                f"Unfilled locations:\n{multiworld.get_unfilled_locations()}.",
```

Reply:
There's a newline at the end of the previous line o:

Reply:
Right, but now after the colon